vcpkg/overlay/sqlite3/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Patch for sqlite3 merged upstream, can be removed here as soon as we bump the submodule version